### PR TITLE
[kjobctl] Embed the CRDs in it's binary

### PR DESCRIPTION
**Pull request by trasc**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:
An admin can use the output of `kjobctl printcrds` to install, update or remove them in/from a cluster without other dependencies.

```bash
  # Install or update the kjobctl CRDs 
  kjobctl printcrds | kubectl apply --server-side -f -
  # Remove the kjobctl CRDs
  kjobctl printcrds | kubectl delete --ignore-not-found=true -f -
```

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: cmd/experimental/kjobctl/pkg/cmd/crds/embed/manifest.gz
Comment (mbobrovskyi) [inline cmd/experimental/kjobctl/pkg/cmd/crds/embed/manifest.gz:1]: This file can be generated differently depending on os or gzip version. IIUC it's nonnecessity to commit this file after build. Can we just ignore it?

Comment (trasc) [reply cmd/experimental/kjobctl/pkg/cmd/crds/embed/manifest.gz:1]: Os independent encoder added.

File: cmd/experimental/kjobctl/docs/commands/kjobctl_crds/_index.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+```
+kjobctl crds [flags]
+```
+
+
+## Examples
+
+```
+  # Install or update the kjobctl CRDs 
+  kjobctl crds | kubectl apply --server-side -f -
```
Comment (mwielgus) [inline cmd/experimental/kjobctl/docs/commands/kjobctl_crds/_index.md:23]: "Crds" without a verb is ambiguous. Can we call it printcrd or alike to indicate that this command just print crds and extra piping is needed?

Comment (trasc) [reply cmd/experimental/kjobctl/docs/commands/kjobctl_crds/_index.md]: done

**Pull request merged**